src/LevelNav.js

```diff
@@ -0,0 +1,55 @@
+import React from "react"
+import './LevelNav.css';
+
+import { Link } from "react-router-dom"
+import {BiChevronLeft, BiChevronRight} from "react-icons/bi"
+
+class LevelNav extends React.Component {
+  render(){
+    let navClass = "LevelNav"
+
+    if(this.props.show) {
+       navClass = 'LevelNav open'
+    }
+
+    let levelNum = this.props.levelNum;
+    if (levelNum === undefined){
+      levelNum = 1;
+    }
+
+    let maxLevel = 4;
+    let leftButton;
+    let rightButton;
+
+    // make sure level number doesn't go out of bounds
+    if(levelNum === 1){
+      leftButton = <a href="#/"> <Link to={`/level/${levelNum}`}> <BiChevronLeft/> </Link> </a>;
+      rightButton = <a href="#/"> <Link to={`/level/${1*levelNum+1}`}> <BiChevronRight/> </Link> </a>;
+    }
+    else if(levelNum === maxLevel){
+      leftButton = <a href="#/"> <Link to={`/level/${levelNum-1}`}> <BiChevronLeft/> </Link> </a>;
+      rightButton = <a href="#/"> <Link to={`/level/${1*levelNum}`}> <BiChevronRight/> </Link> </a>;
+    }
+    else {
+      leftButton = <a href="#/"> <Link to={`/level/${levelNum - 1}`}> <BiChevronLeft/> </Link> </a>;
+      rightButton = <a href="#/"> <Link to={`/level/${1*levelNum + 1}`}> <BiChevronRight/> </Link> </a>;
+    }
```

Comment:
You can avoid some duplication here by calculating prevLevelNum and nextLevelNum in the if-statement, then passing those to the leftButton and rightButton elements when you render them!

Also I'm just wondering, how come you put the Links in an anchor element that links to '#/'? I think this is just something I'm not familiar with